src/spawn_sync.cc

```diff
@@ -708,9 +708,7 @@ Local<Object> SyncProcessRunner::BuildResultObject() {
 
   if (term_signal_ > 0)
     js_result->Set(context, env()->signal_string(),
-                   String::NewFromUtf8(env()->isolate(),
-                                       signo_string(term_signal_))
-                       .ToLocalChecked())
+                   OneByteString(env()->isolate(), signo_string(term_signal_)))
```

Comment:
Should we cache this signo_string() result in the future?

Reply:
I wouldn't think that is necessary.